testing/tests/interop/quic-go_quic-go/zerortt.zeek

```diff
@@ -0,0 +1,8 @@
+# @TEST-REQUIRES: zeek -b -e 'print PacketAnalyzer::ANALYZER_PPP ==  PacketAnalyzer::ANALYZER_PPP'
```

Comment:
Similar pattern already exists in `handshake.zeek`, could you document why this is needed? With that I'd know why this test doesn't run with zeek-5.2.

Also for new tests a brief `@TEST-DOC` would be nice (or a comment).

Here and below.